system-test/test.sh

```diff
@@ -7,7 +7,8 @@ retry() {
 }
 
 function timeout_after() {
-  if [ -f /bin/busybox ]; then
+  # timeout on Node 11 alpine image requires -t to specify time.
+  if [ -f /bin/busybox ] &&  [[ $(node -v) =~ ^v11.* ]]; then
```

Comment:
Nit: let's use either double brackets or single for both? But I'm approving the change to stop the alerts.

Reply:
I think the double and single brackets have slightly different semantics (I need double brackets for the second condition; I'm not sure if I can use them for the first).

I'll look into addressing this in a follow-up.